package.json

```diff
@@ -32,8 +32,11 @@
   "dependencies": {
     "canonical-json": "0.0.4",
     "csv-parse": "^3.1.3",
+    "fs-extra": "^8.0.1",
```

Comment:
Is this new dependency required? Have you seen the `lib/sync-fs` which has function `writeJson`?